### PR TITLE
docs: add adaptive jitter buffer from rem

### PR DESCRIPTION
**Pull request by cspiel1**

### CODE REVIEW

File: docs/aubuf/config [hunk 1]
```diff
@@ -0,0 +1,56 @@
+#audio_path		/usr/local/share/baresip
+audio_player		pulse,
```
Comment (alfredh) [inline docs/aubuf/config:2]: libre and baresip are 2 different projects.

Baresip knows about Libre, but libre does not know about any application using it (e.g. baresip).

Please dont put references to `baresip` inside the libre project.

**Comment by alfredh** [comment]
I would suggest to move some of the content here to the Wiki.

Perhaps content that is dynamic by nature ...

**Comment by cspiel1** [comment]
README.md contains
- description of the adaptive jitter buffer algorithm in `aubuf`
- description how to test the jitter buffer and generate plots

`docs/aubuf` contains also scripts for generating the plots. Baresip is used as generation tool.

So the content belongs to `aubuf`. Thus maybe should be moved to the re wiki and not baresip wiki. Okay?

**Comment by cspiel1** [comment]
I move the README.md to https://github.com/baresip/re/wiki/Adaptive-jitter-buffer.

The scripts for generating the ajb plots and the config therefor should stay in the git repository. Also I wouldn't separate the config from the scripts. Should I move it to baresip? In which directory?

File: docs/aubuf/generate_plots.sh [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+i=1
+for ptime in 20 10 5 15 30 40; do
+
+    sed -e "s/ptime=[0-9]*/ptime=$ptime/" -i accounts
+    for buf in $(( ptime + ptime/2 )) $(( ptime )) $(( 2*ptime )) $(( 4*ptime )) $(( 6*ptime )); do
+        echo "########### ptime $ptime buffer $buf ###############"
+
+        sed -e "s/audio_buffer\s*[0-9]*\-.*/audio_buffer   $buf-250/" -i config
+        baresip -v -f . > /tmp/b.log 2>&1 &
```
Comment (alfredh) [inline docs/aubuf/generate_plots.sh:59]: any reference to "baresip" should not live in this repo.

perhaps "baresip/tools" or "baresip/test" is better.

**Comment by alfredh** [comment]
have a brief documentation is fine.

if you have a large piece of documentation, it will most likely get out of sync, when the
source code is updated.

File: docs/aubuf/generate_plots.sh (hunk 2 above)
Comment (cspiel1) [reply docs/aubuf/generate_plots.sh:59]: I'll move the script and its dependencies to `baresip/tools/aubuf`.

**Comment by cspiel1** [comment]
> have a brief documentation is fine.
> 
> if you have a large piece of documentation, it will most likely get out of sync, when the source code is updated.

Right, like everything in the Wiki. In this case https://github.com/baresip/re/wiki/Adaptive-jitter-buffer strongly is related to the code. With Doxygen it might be better than with a Wiki page. It also supports $\LaTeX$ formulas. And I think the documentation of the ajb algorithm is helpful and should not be dropped without replacement. Maybe I'll try it with Doxygen?

**Comment by alfredh** [comment]
what is the next step here ?

**Comment by cspiel1** [comment]
I'll try the mentioned steps these days.
- move scripts to `baresip/tools/aubuf`
- move wiki page to Doyxgen in `re/rem/src/aubuf/ajb.c`

**Comment by cspiel1** [comment]
replaced by: https://github.com/baresip/baresip/pull/2641

**Pull request closed without merging**